<launch>
    <node pkg="odom_estimate" type="uav_pose_predictor" name="uav_pose_predictor" output="screen">
        <rosparam command="load" file="$(find messages)/config/predictor.yaml" />
        <!-- subscribes from mavros -->
        <remap from="/current/pose" to="/mavros/local_position/pose" />
        <remap from="/current/velocity" to="/mavros/local_position/velocity_body/twist/linear" />
        <remap from="/current/acceleration" to="/mavros/imu/data/linear_acceleration" />
        <!-- publishes -->
        <remap from="/predicted/pose/theta" to="/predicted/spherical_pose/theta" />
        <remap from="/predicted/pose/phi" to="/predicted/spherical_pose/phi" />
    </node>
</launch>